operators/samples/templates.yaml

```diff
@@ -73,8 +73,7 @@ spec:
       containerStartupOptions:
         contentPath: /config/workspace
         startupArgs:
-          - "false"
-          - "proxy"
+          - "true"
```

Comment:
Would personally make this argument more self explanatory, like `--disable-marketplace`, otherwise it is unclear its meaning.

Reply:
`for ARGUMENT in "$@"; do
    key=$(echo "$ARGUMENT"|cut -d = -f 1)
    value=$(echo "$ARGUMENT"|cut -d = -f 2)

    if [ "$key" == "--marketplace-disabled" ] && [ "$value" == "true" ]; then
        export EXTENSIONS_GALLERY='{"serviceUrl": ""}'
    fi
done`

Could be a good idea to add something like that inside the container entry point or do you know any better solutions?

Reply:
Would just use `--disable-marketplace` (without true/false) to avoid the complexity of spitting the argument, but as you prefer.